src/sidebar.js

```diff
@@ -398,6 +404,7 @@ Sidebar.propTypes = {
 
 Sidebar.defaultProps = {
   docked: false,
+  sidebarId: '',
```

Comment:
This will give the sidebar an empty id. I think it's best to not set a default prop for sidebarId at all.